systems/actions.md

```diff
@@ -93,7 +96,7 @@ Your character moves at a comfortable pace, moving up to their normal movement.
 `secondary` Your character draws a weapon.
 
 ## Attack of opportunity
-`reaction` An attack of opportunity is be triggered by a foe performing an unsafe action while within your weapon reach.
+`reaction` An attack of opportunity is be triggered by a foe performing an `unsafe` action while within your weapon reach.

```

Comment:
So we have something somewhere about the reaction being a choice? Don't want to make it seem like it has to be triggered